static.json

```diff
@@ -0,0 +1,12 @@
+{
+  "root": "build/",
+  "routes": {
+    "/**": "index.html"
+  },
+  "https_only": true,
+  "headers": {
+    "/**": {
+      "Strict-Transport-Security": "max-age=31557600"
```

Comment:
This resolves to a cache lifetime of 8 hours. So if we ever need to turn this off, that's how long we'll have to wait for a returning visiter's browser to stop forcing https. We shouldn't ever have to turn this off though...unless we ever move off Heroku.